src/sys/windows.rs

```diff
@@ -42,6 +42,11 @@ const WSA_FLAG_OVERLAPPED: DWORD = 0x01;
 
 pub use winapi::ctypes::c_int;
 
+/// Fake MSG_TRUNC flag for the [`RecvFlags`] struct.
+///
+/// Value copied from Linux.
```

Comment:
I think you can just say the value is defined by us and just use `0b1`.

Reply:
Done :)